packages/expo-modules-core/common/cpp/fabric/ExpoViewComponentDescriptor.cpp

```diff
@@ -2,6 +2,13 @@
 
 #include "ExpoViewComponentDescriptor.h"
 
+
+#if REACT_NATIVE_TARGET_VERSION >= 73 && IS_NEW_ARCHITECTURE_ENABLED
+// Android can't find the definition of these static fields
+bool facebook::react::CoreFeatures::enableDefaultAsyncBatchedPriority;
+bool facebook::react::CoreFeatures::enablePropIteratorSetter;
```

Comment:
that's interesting to me. we had 0.73 on react-native-modules but not seeing this error before. would take a look further.

Reply:
Yeah, I'm not sure why this is happening but Reanimated is facing a similar problem 

https://github.com/expo/expo/blob/70bee13f4e10a09d20f394f689a3b9c5a8dc3dd3/ios/vendored/unversioned/react-native-reanimated/Common/cpp/NativeModules/NativeReanimatedModule.cpp#L39

Reply:
ok for this one. the correct fix should be
```diff
--- a/packages/expo-modules-core/android/src/fabric/CMakeLists.txt
+++ b/packages/expo-modules-core/android/src/fabric/CMakeLists.txt
@@ -43,6 +43,7 @@ target_link_libraries(fabric
   ReactAndroid::react_render_debug
   ReactAndroid::react_render_graphics
   ReactAndroid::react_render_mapbuffer
+  ReactAndroid::react_utils
   ReactAndroid::rrc_view
   ReactAndroid::runtimeexecutor
   ReactAndroid::yoga
```

Reply:
Ohh thanks for finding a proper fix for this, updated